Gemfile

```
@@ -37,15 +37,15 @@ gem 'sass-rails', '>= 6'
 # Anchored versions, do not change
 gem 'puma', '~> 4.3.8'
 gem 'puma-plugin-statsd', '~> 1.2.1'
-gem 'rails', '~> 6.0.3'
+gem 'rails', '~> 6.1.3.2'
```

```suggestion
gem 'rails', '~> 6.1.3'
```

I'd recommend locking just to the patch.